src/main/groovy/com/codebrig/arthur/observe/structure/filter/operator/relational/RelationalOperatorFilter.groovy

```diff
@@ -58,4 +61,52 @@ class RelationalOperatorFilter extends StructureFilter<RelationalOperatorFilter,
         }
         return false
     }
+
+    static SourceNode getLeftOperand(SourceNode node) {
```

Comment:
It looks like there will be a conflict here and https://github.com/CodeBrig/Arthur/blob/c3fdd43e3cc4c758bee987ec140a8f51ec03251b/src/main/groovy/com/codebrig/arthur/observe/structure/filter/operator/relational/RelationalOperatorFilter.groovy#L66. Please update them so they will not conflict on merge.

Reply:
By the conflict, I'm talking about the `getSendOperatorLeft` vs `getPropOperand1`.

Reply:
@BFergerson These two method groups should actually be added and merged together in `RelationalOperatorFilter.groovy`, but they're in different branches/issues:

- `getPropOperand1` / `getPropOperand1` (#40)
- `getSendOperatorLeft` / `getSendOperatorRight` (#90)

Should I added them in `RelationalOperatorFilter.groovy` for both branches, or what do you suggest?

Reply:
@chess-equality, I understand that both `getPropOperand1/getSendOperatorLeft` (etc) are needed but the implementing function `getLeftOperand` is different between the two branches.

It looks like `getLeftOperand` will need to switch based off language. As for the merging, you can copy + paste the same code in both branches.

I hope that makes sense.

Reply:
@BFergerson Please see my commits:

[[#40] Added getSendOperatorLeft / getSendOperatorRight](https://github.com/CodeBrig/Arthur/pull/86/commits/722547f02cbce050590d0d18ab29124c39ef829f)

[[#90] Added getPropOperand1 / getPropOperand2](https://github.com/CodeBrig/Arthur/pull/93/commits/1ce63d18e67b0d0f2782adbd716bfb9b99c15f34)

C++ branch is not yet merged so I did not use language switches here.